### PR TITLE
add sections to README

### PR DESCRIPTION
**Pull request by jasonburkert-ccpo**
Adds sections:
- Install Prerequisites (includes table and commands)
- Install JavaScript Dependencies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,24 @@
...
+| ----------- | ----------- |----------- |
+| Node Version Manager | `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh \| bash` | Then restart your shell. Details available in the [nvm README](https://github.com/nvm-sh/nvm#installing-and-updating). |
+| Node.js | `nvm install` | Node.js version is specified in file `./.nvmrc` |
+| TypeScript | `npm install -g typescript` | [Documentation](https://www.typescriptlang.org/docs/) |
+| AWS CDK Toolkit | `npm install -g aws-cdk` | [Developer Guide](https://docs.aws.amazon.com/cdk/latest/guide/cli.html) |
+
+### Install JavaScript Dependencies 
+```
+npm ci
+npm run bootstrap
```
Comment (jasonburkert-ccpo) [inline README.md:20]: Need to install first otherwise no lerna for bootstrap

Comment (jasonburkert-ccpo) [reply README.md:21]: This is assuming running commands from root.

Comment (kylelaker-ccpo) [reply README.md:21]: I think that is a perfectly fair assumption

**Review by kylelaker-ccpo — APPROVED** [review]
This is all great! You had mentioned that we may need to add some info on configuring the AWS profiles. I agree but I think we should hold off until we have a better idea of what "final" looks like there for us

**Pull request merged**